src/tests/ftest/io/EightServers_mpiio.yaml

```diff
@@ -0,0 +1,69 @@
+hosts:
+   test_machines:
+        test_servers:
+            - boro-A
+            - boro-B
+            - boro-C
+            - boro-D
+            - boro-E
+            - boro-F
+            - boro-G
+            - boro-H
+        test_clients:
+            - boro-I
+            - boro-J
+            - boro-K
+            - boro-L
+            - boro-M
+            - boro-N
+            - boro-O
+            - boro-P
+timeout: 2000
+server:
+    server_group: daos_server
```

Comment:
Can you please change this from "server_group" to "name" so that when the Control Plane patch lands it works without any changes

Reply:
Done.